Code/GraphMol/Depictor/catch_tests.cpp

```diff
@@ -227,6 +227,27 @@ TEST_CASE("use ring system templates") {
   TEST_ASSERT(RDKit::feq(diff.length(), 1.0, .1))
 }
 
+TEST_CASE("templates are aware of E/Z stereochemistry") {
+  // this is a molecule we have a template for
+  auto mol1 =
+      ""
```

Comment:
it's not clear why this empty string is here, please remove it?

Reply:
removed